components/developer/gcc-6/patches/use-cxa_atexit.patch

```diff
@@ -1,11 +0,0 @@
---- gcc-6.4.0/gcc/config.gcc.~1~	2017-01-10 00:01:26.582656000 +0000
```

Comment:
Is this patch no longer necessary?

Reply:
It is not necessary since https://github.com/OpenIndiana/gcc/blob/il-6_5_0/gcc/config.gcc#L853